app.js

```diff
@@ -19,13 +20,14 @@ app.use(async function(ctx, next) {
 
         switch (e.status) {
         case 400:
+        case 401:
         case 403:
         case 404:
             ctx.throw(e);
             break;
         default:
             // TODO: Figure out which errors should be exposed to user
-            console.error('Error: ', e, JSON.stringify(e));
+            // console.error('Error: ', e, JSON.stringify(e));
```

Comment:
why comment out?

Reply:
was testing, uncommented in upcoming commit

Reply:
hmm, seems still commented?

Reply:
sorry I was seeing the console errors in testing and it was confusing me as to which tests were not passing so for convenience commented out again while finishing the review edits.

uncommented again, tests passing